lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -829,17 +899,29 @@ def remove_toolitem(self, name):
                     self._groups[group].remove(toolitem)
         del self._toolitems[name]
 
+    def set_flow(self, flow):
```

Comment:
Also, `flow`.... why not `orientation` or `direction`.
How about to pass it as argument to `__init__` with a default `horizontal`? it shouldn't be changeable after creation